### PR TITLE
feat: add templ

### PR DESCRIPTION
**Pull request by papierkorp**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is ...

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
Looks like you reformatted the entire json file, please don't 😅 
The package itself looks good though.

**Comment by papierkorp** [comment]
Sorry i havent noticed that I accidentally formatted the whole file >.<

**Comment by braver** [comment]
Ah, that's better. Looks like the tests require indentation to be in tabs though.

**Comment by papierkorp** [comment]
done :D

File: repository/t.json [hunk 1]
```diff
@@ -540,6 +540,18 @@
 				}
 			]
 		},
+		{
+			"name": "Templ (go)",
+			"details": "https://github.com/papierkorp/sublime-templ",
+			"labels": ["templ", "golang", "go"],
+			"description": "Syntax highlight for the go templ Plugin",
```
Comment (braver) [inline repository/t.json:547]:
```suggestion
```

You don't need this line, Package Control will take the repo's description.

**Review by braver — CHANGES_REQUESTED** [review]

**Review by braver — APPROVED** [review]

**Pull request merged**